extensions/ql-vscode/src/distribution.ts

```diff
@@ -297,6 +297,10 @@ class ExtensionSpecificDistributionManager {
         `but encountered an error: ${e}.`);
     }
 
+    // NOTE: We are tacitly assuming that at this point
```

Comment:
I thought this is no longer true?

Reply:
After some consideration, I felt persuaded that your argument that downloading assets serially was bad enough to warrant not attempting to go so far out of my way to be future proof here. This PR is trying to be the minimal step forward that at least accounts for the change in cli distribution that we *do* expect to make.